include/openenclave/internal/eeid.h

```diff
@@ -202,10 +198,9 @@ oe_result_t oe_eeid_ntoh(
 
 typedef struct
 {
-    size_t sgx_evidence_size;     /* Size of base-image evidence */
-    size_t sgx_endorsements_size; /* Size of base-image endorsements */
-    size_t eeid_size;             /* Size of EEID */
-    uint8_t data[];               /* Data (same order as the sizes) */
+    size_t sgx_evidence_size; /* Size of SGX evidence */
+    size_t eeid_size;         /* Size of EEID */
+    uint8_t data[];           /* Data (same order as the sizes) */
 } oe_eeid_evidence_t;
```

Comment:
looks like this name was already bad... sgx fields in a non-sgx struct.   give that this is in an internal header file, we should just fix it.

Reply:
Unnecessarily so; it's the base-image evidence size, Fixed.